packages/eyes/src/generate-snapshots/build-base-files.ts

```diff
@@ -0,0 +1,45 @@
+import {consoleLog} from '@ui-autotools/utils';
+import fs from 'fs';
+import path from 'path';
+import {generateFilename, generateData} from './filename-utils';
+import { IRegistry } from '@ui-autotools/registry';
+
+export const buildBaseFiles = (projectPath: string, Registry: IRegistry) => {
+  consoleLog('Building base files...');
+  consoleLog('Importing Metafiles...');
+
+  const autotoolsFolder = path.join(projectPath, '.autotools', 'tmp');
+
+  if (!fs.existsSync(autotoolsFolder)) {
+    fs.mkdirSync(autotoolsFolder);
+  }
+
+  const stylePathPrefix = '../../'; // We're two folders deep in .autotools
+  const compPathPrefix = '../../';
+
+  Registry.metadata.components.forEach((componentMetadata) => {
+    const simIndex = componentMetadata.simulations.length;
+    const styles = componentMetadata.styles;
+    const compPath =  path.join(compPathPrefix, componentMetadata.path);
+    const compName = componentMetadata.exportName;
+
+    if (compName) {
+      for (let i = 0; i < simIndex; i++) {
+        const simulationName = componentMetadata.simulations[i].title;
+        if (styles.size) {
+          styles.forEach((style) => {
+            const stylePath = path.join(stylePathPrefix, style.path);
```

Comment:
Use `projectPath`. `stylePath` is relative to the current folder, and assumes that the current folder is the project folder.